### PR TITLE
add image()

### PR DESCRIPTION
**Pull request by ladyada**
for linux/PIL

### CODE REVIEW

**Comment by brentru** [comment]
Tested working, `ssd1306_framebuftest.py` and `framebuf_simpletest.py` look good on Pi 3B+.

**Comment by ladyada** [comment]
@brentru please go thru the 'review' process - its good practice! :)

File: adafruit_framebuf.py [hunk 1]
```diff
@@ -282,8 +282,7 @@ def image(self, img):
         # Clear buffer
         for i in range(len(self.buf)):
             self.buf[i] = 0
-        # Iterate through the memory pages
-        index = 0
+        # Iterate through the pixels
         for x in range(self.width):       # yes this double loop is slow,
```
Comment (brentru) [inline adafruit_framebuf.py]: Possibly change in the nested loop, `imwidth` should be valid as they're checked above
`self.width` to `imwidth`

File: adafruit_framebuf.py [hunk 2]
```diff
@@ -282,8 +282,7 @@ def image(self, img):
...
             for y in range(self.height):  #  but these displays are small!
```
Comment (brentru) [inline adafruit_framebuf.py]: and `self.height` to `imheight`

File: adafruit_framebuf.py (hunk 1 above)
Comment (ladyada) [reply adafruit_framebuf.py]: yes they must be identical

**Pull request merged**